src/layouts/about.jsx

```diff
@@ -2,7 +2,186 @@ import React from 'react';
 
 const About = () => (
   <div className="about">
-    <h2>About us!</h2>
+    <div className="group-photo-container text-center">
+      <img className="img-responsive group-photo" src="/images/about-cover.png" alt="A group discussion about Signsfive" />
+    </div>
+
+
+    <div className="container">
+      <section className="container py-5 px-5">
+        <h3 className="tracked pb-2">About SignsFive</h3>
+        <p>
+          <strong>
+            Make the search and share for STE(A)M (Science, Technology,
```

Comment:
Make it one line, no broken line here

Reply:
`p` and `strong` in the same line?

Reply:
i meant line 17 and 18 `Make the search and share for STE(A)M (Science, Technology, Engineering, <em>Art</em>, Math) signs easier.`
instead of..
`Make the search and share for STE(A)M (Science, Technology, (break line here)
Engineering, <em>Art</em>, Math) signs easier.`